### PR TITLE
TFW-3451 - added base combo bar line chart without z axis

### PR DESCRIPTION
**Pull request by troyblank**
This adds anything for combo bar line chart not included as feature enhancements. Which is basically a bar chart using the line chart demo data :) 

Screen Shot 2019-04-30 at 3 34 59 PM

There is a small coverage issue here that is fixed with the PR jason is doing > https://github.com/topbox-rci/rci-components/pull/1101/commits/a74fc16e5c7d80ce072a2294ebe16a2fd3e96750

so please ignore that.

### CODE REVIEW

File: components/visualizations/comboBarLine/comboBarLine.js [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+        const { isPending } = this.props;
+
+        this.setContentWidth();
+        this.buildXAxis();
+        this.buildYAxis();
+        this.buildSVG();
+
+        if (!isPending) {
+            // apply data to graph if data should be available
+            // this.maxDataLength = data.length;
```
Comment (jasonhale) [inline components/visualizations/comboBarLine/comboBarLine.js]: Commented code.  Needed?

File: components/visualizations/comboBarLineChart.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            PropTypes.object
+        ),
+        metaData: PropTypes.shape({
+            yKey: PropTypes.string,
+            xKey: PropTypes.string
+        })
+    };
+
+    state = {
+        xAxisMode: 'day'
```
Comment (jasonhale) [inline components/visualizations/comboBarLineChart.js]: we have a constant for that now.  :)

File: components/visualizations/comboBarLineChart.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        return null;
+    }
+
+    render() {
+        const { label, id, metaData, isPending = false, data } = this.props;
+        const { xAxisMode } = this.state;
+        let filteredData, meta;
+
+        if (!isPending) {
+            filteredData = data;
```
Comment (jasonhale) [inline components/visualizations/comboBarLineChart.js]: This line doesn't look necessary.

Comment (troyblank) [reply components/visualizations/comboBarLineChart.js]: nope it's not, removed!

File: components/visualizations/comboBarLineChart.js (hunk 2 above)
Comment (troyblank) [reply components/visualizations/comboBarLineChart.js]: haha, no fair no fair, updated!

File: components/visualizations/comboBarLine/comboBarLine.js (hunk 1 above)
Comment (troyblank) [reply components/visualizations/comboBarLine/comboBarLine.js]: noooope, removed thanks.

**Pull request merged**